packages/block-library/src/embed/embed-placeholder.native.js

```diff
@@ -64,9 +65,24 @@ const EmbedPlaceholder = ( {
 				<View style={ containerStyle }>
 					<View style={ styles.embed__icon }>{ icon }</View>
 					<Text style={ labelStyle }>{ label }</Text>
-					<Text style={ styles[ 'embed-empty__description' ] }>
-						{ __( 'ADD LINK' ) }
-					</Text>
+					{ cannotEmbed ? (
+						<>
+							<Text style={ labelStyle }>
+								{ __(
+									'Sorry, this content could not be embedded.'
+								) }
+							</Text>
+							<Text
+								style={ styles[ 'embed-empty__description' ] }
+							>
+								{ __( 'EDIT LINK' ) }
+							</Text>
+						</>
+					) : (
+						<Text style={ styles[ 'embed-empty__description' ] }>
+							{ __( 'ADD LINK' ) }
+						</Text>
+					) }
```

Comment:
In an incoming PR, I will replace this part with a new component `EmbedNoPreview`.